integration_tests/src/main/python/string_test.py

```diff
@@ -314,9 +314,6 @@ def test_length():
                 'CHAR_LENGTH(a)',
                 'CHARACTER_LENGTH(a)'))
 
-# Once the xfail is fixed this can replace test_initcap_space
-@incompat
-@pytest.mark.xfail(reason='https://github.com/NVIDIA/spark-rapids/issues/120')
 def test_initcap():
```

Comment:
This test should include more characters than just `[aAbB]`.  Failing to test a wider character range in the words is what led to #2786 being missed during testing.

Reply:
Updated